versioned_docs/version-2.0-2.4/installation/resources/chart-options/chart-options.md

```diff
@@ -2,4 +2,8 @@
 title: Rancher Helm Chart Options
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/helm-chart-options"/>
```

Comment:
Need to add to the corresponding 2.5, 2.6, /docs pages.

Reply:
These were actually already addressed in https://github.com/rancher/rancher-docs/pull/604